src/packages/core/tests/server.test.ts

```diff
@@ -336,15 +336,71 @@ describe("server", () => {
 
       for (const specificPort of validPorts) {
         s = Ganache.server(defaultOptions);
+        assert.strictEqual(
+          s.address(),
+          null,
+          "server.address() should be null before `server.listen(...)` is called"
+        );
         await s.listen(<any>specificPort);
 
         try {
+          const address = s.address();
+          assert.notEqual(
+            address,
+            null,
+            "server.address() should not be null after listening"
+          );
+          assert.strictEqual(
+            address!.port,
+            +specificPort,
+            "server.address().port should be non-zero after server.listen(0)"
```

Comment:
```suggestion
            "server.address().port should be equal to the specified port after server.listen(nonZeroPort)"
```
Or something like that, since this test isn't testing `server.listen(0)`